build.sbt

```diff
@@ -1,5 +1,15 @@
+import sbtcrossproject.{crossProject, CrossType}
 
-crossScalaVersions := Seq("2.10.6", "2.11.8", "2.12.0")
+lazy val testAllCommand = Command.command("testall"){state =>
```

Comment:
you can also do

.nativeSetting(
  crossScalaVersions --= Seq("2.10.6", "2.12.2")
)